js/popup-window.js

```diff
@@ -0,0 +1,156 @@
+        </div>
+        <div class="spans-popup">
+          <span class="client">Canopy</span>
+          <span><i class="fa fa-circle fa-xs"></i></span>
+          <span class="role"> Back End Dev</span>
+          <span><i class="fa fa-circle"></i></span>
+          <span class="year">2015</span>
+        </div>
+        <img src=${cardetails[i].img} alt="Snapshoot-Portfolio" class="img-popup" id="popupimage">
+        <img src=${cardetails[i].img} alt="mobile-popup" class="img-popup-mobile" id="popupimage">
+        <div class="modal-container"> 
+          <div class="information-popup">
+            <p class="paragraph-popup">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              <br>
+              <br>
+              <br>
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+            </p>
+            <p class="paragraph-popup-mobile">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+            </p>
+          </div>
+          <div class="tags-button">
+            <ul class="tags">
+              <li class="tag">${cardetails[i].technologies[0]}</li>
+              <li class="tag">${cardetails[i].technologies[1]}</li>
+              <li class="tag">${cardetails[i].technologies[0]}</li>
+            </ul>
+            <ul class="tags-hidden">
+              <li class="tag">Ruby</li>
+              <li class="tag">Bootstrap</li>
+            </ul>
+            <hr>
+            <div class="btn-block">
+              <a href="https://joaquin123455.github.io/microverse-portfolio/" target="_blank" rel="noopener" class="btn-a">
+              <button class="btn-popup-left" type="button"> See Live <img class="img-btn"
+                  src="./img/live-popup.svg" alt="Button of project "> </button></a>
+              <a href="https://github.com/Joaquin123455" target="_blank" rel="noopener" class="btn-a">
+              <button class="btn-popup-right" type="button">See Source <img class="img-btn"
+                  src="./img/github-popup.svg" alt="Button of repository"> </button></a>
```

Comment:
[OPTIONAL] Again this optional and recommended requirement kindly don't skip it as I mentioned above :point_up_2: please stick to the Figma design your `mobile version` buttons are not the same as the Figma design. Check Below :point_down:  the comparison between your design and Figma design :+1:  

**Your Design**
![Screenshot from 2023-04-19 01-29-46](https://user-images.githubusercontent.com/45226848/232901021-7ae3b9e9-583e-4466-9b5c-d3e8ad87a9fe.png)

**Figma design**

![Screenshot from 2023-04-19 01-30-47](https://user-images.githubusercontent.com/45226848/232901032-66a1fcd0-4cd2-42ab-88e2-15c3aed47087.png)